ubuntudesign/landscape.canonical.com
# added ctas for buy.u.c

## Done

* added buttons to the new buy.u.c site on the homepage

![image](https://cloud.githubusercontent.com/assets/441217/17854677/2fe68e80-686c-11e6-8571-3f49ad6a3c54.png)

* added a link to buy in the footer of every page

## QA

1. go to the homepage and look at the 'Get Landscape, world-class support and more with Ubuntu Advantage' section and see three buttons now
2. go to the footer and see the link to 'Buy Ubuntu Advantage'

## Issue / Card

[marketing request](https://docs.google.com/document/d/1SHFbVbgG340KpP3L45PtZ9P0Qx1ek9wZ1q7_t0wgpM0/edit#)
